### PR TITLE
Adding my location

### PR DESCRIPTION
**Pull request by Rushaabh**
* Making an unordered list.
* @githubteacher 
This pull request closes #927

### CODE REVIEW

**Comment by imiro** [comment]
Looks awesome, @Rushaabh! 🇮🇳 🎉

File: _pins/Rushaabh.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
```
Comment (crichID) [inline _pins/Rushaabh.json]: @Rushaabh  the reason the tests are failing is that you have a space at the beginning of each line. If you click the Files changed tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

File: _pins/Rushaabh.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ githubHandle: Rushaabh.
```
Comment (crichID) [inline _pins/Rushaabh.json]: @Rushaabh you should also remove the period `.` at the end of your github username.

File: _pins/Rushaabh.json (hunk 1 above)
Comment (Rushaabh) [inline _pins/Rushaabh.json]: Thank you for pointing out. 👍

**Comment by crichID** [comment]
@Rushaabh I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3040/commits/83618526623be8323c2123a12579bf54b399c3dd. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**